src/Arcus.Observability.Correlation/Arcus.Observability.Correlation.csproj

```diff
@@ -17,11 +17,12 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
-    <FrameworkReference Include="Microsoft.AspNetCore.App" />
+  <ItemGroup Condition="'$(TargetFramework)' != 'netstandard2.0'">
```

Comment:
If the targetframework is netstandard1.0, this will also evaluate to true ....   Maybe I'm too much of a nitpicker :)
But ok, I guess we'll have another look at it via another PR.  (And the supported frameworks are now only netstandard2.0 and netcoreapp3.1, so then it's not a problem I guess.